src/Octoshift/GithubApi.cs

```diff
@@ -341,6 +341,34 @@ public virtual async Task<string> GetMigrationFailureReason(string migrationId)
             return (string)data["data"]["node"]["failureReason"];
         }
 
+        public virtual async Task<string> GetMigrationLogUrl(string orgLogin, string repositoryName)
```

Comment:
every other function in here uses the arg name "org" not org login. can we make it consistent.
```suggestion
        public virtual async Task<string> GetMigrationLogUrl(string org, string repositoryName)
```

Reply:
Updated in 969e878dcc0f15a5fd2569cb296ca01f979dc6ef!